### ISSUE TITLE
Language Packs category is for display language extensions

### ISSUE THREAD

**Issue by gregvanl** [prismagraphql/vscode-graphql#41]
The "Language Packs" VS Code extension category is to designate disaplay language (German, Bulgarian) extensions and not for general programming language extensions.

Please remove "Language Packs" from the categories in package.json:

```json
"categories": [
    "Programming Languages",
    "Snippets",
    "Linters",
    "Themes",
    "Other",
    "Language Packs" <--
```

**Comment by gregvanl** [comment]
As you can see below, the GraphQL extension looks out of place in the "Language Packs" query:

![image](https://user-images.githubusercontent.com/12818376/42003734-ce86156a-7a21-11e8-924d-c47b7d46ab4c.png)

**Comment by divyenduz** [comment]
@gregvanl Thanks for raising the issue and pointing it out. Fixed here: https://github.com/prismagraphql/vscode-graphql/pull/42

I will cut a release today to market place.

**Comment by divyenduz** [comment]
Released in `0.0.10` 🎉

**Comment by gregvanl** [comment]
Thanks!
